### PR TITLE
Rds backup

### PR DESCRIPTION
**Pull request by irina-pereiaslavskaia**

### CODE REVIEW

**Comment by irina-pereiaslavskaia** [comment]
recheck

**Comment by irina-pereiaslavskaia** [comment]
recheck

**Comment by irina-pereiaslavskaia** [comment]
recheck

**Comment by irina-pereiaslavskaia** [comment]
recheck

File: tests/integration/targets/rds_backup/tasks/main.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        name: "test_ansible_module"
+      register: rds_backup
+      ignore_errors: yes
+
+    - name: assert result
+      assert:
+        that:
+          - rds_backup is success
+          - rds_backup is not changed
+          - 'rds_backup.msg == "RDS instance test-db does not exist"'
```
Comment (anton-sidelnikov) [inline tests/integration/targets/rds_backup/tasks/main.yaml:18]: end line, and add step in with check_mode: true

Comment (irina-pereiaslavskaia) [reply tests/integration/targets/rds_backup/tasks/main.yaml]: Done

**Review by anton-sidelnikov — APPROVED** [review]

**Review by anton-sidelnikov — APPROVED** [review]

File: plugins/modules/rds_backup.py [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DOCUMENTATION = '''
+---
+module: rds_backup
+short_description: Manage RDS backup
+extends_documentation_fragment: opentelekomcloud.cloud.otc
+version_added: "0.0.1"
```
Comment (gtema) [inline plugins/modules/rds_backup.py:19]: version too old - use 0.2.0

File: plugins/modules/rds_backup.py [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+        sample: "COMPLETED"
+      type:
+        description: Indicates the backup type.
+        type: str
+        sample: "manual"
+'''
+
+EXAMPLES = '''
+#Create RDS backup (only required parameters are specified)
+- rds_backup:
```
Comment (gtema) [inline plugins/modules/rds_backup.py:125]: use FQCN names in examples

File: plugins/modules/rds_backup.py [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+    default: present
+    type: str
+  wait:
+    description:
+      - If the module should wait for the RDS backup to be created or deleted.
+    type: bool
+    default: 'yes'
+  timeout:
+    description:
+      - The amount of time the module should wait.
```
Comment (gtema) [inline plugins/modules/rds_backup.py:58]: I would say "Duration in seconds ..."

**Review by gtema — CHANGES_REQUESTED** [review]

File: plugins/modules/rds_backup.py (hunk 3 above)
Comment (irina-pereiaslavskaia) [reply plugins/modules/rds_backup.py]: Done

File: plugins/modules/rds_backup.py (hunk 4 above)
Comment (irina-pereiaslavskaia) [reply plugins/modules/rds_backup.py]: Done

File: plugins/modules/rds_backup.py (hunk 2 above)
Comment (irina-pereiaslavskaia) [reply plugins/modules/rds_backup.py]: Done

**Review by anton-sidelnikov — DISMISSED** [review]

File: tests/integration/targets/rds_backup/tasks/main.yaml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+---
+- module_defaults:
+    rds_backup:
+      cloud: "{{ test_cloud }}"
+  block:
+    - name: Get error message that rds instance is absent
+      rds_backup:
```
Comment (anton-sidelnikov) [inline tests/integration/targets/rds_backup/tasks/main.yaml:7]: use fqdn name here for clients like opentelekomcloud.cloud.rds_backup

File: plugins/modules/rds_backup.py [hunk 6]
```diff
@@ -0,0 +1,289 @@
...
+        type: int
+        sample: 220276
+      status:
+        description: Indicates the backup status.
+        type: str
+        sample: "COMPLETED"
+      type:
+        description: Indicates the backup type.
+        type: str
+        sample: "manual"
```
Comment (anton-sidelnikov) [inline plugins/modules/rds_backup.py:120]: use here 2 spaces instead of 4

Comment (irina-pereiaslavskaia) [reply plugins/modules/rds_backup.py]: Done

File: tests/integration/targets/rds_backup/tasks/main.yaml (hunk 5 above)
Comment (irina-pereiaslavskaia) [reply tests/integration/targets/rds_backup/tasks/main.yaml]: Done

**Comment by irina-pereiaslavskaia** [comment]
recheck

**Comment by lego963** [comment]
recheck

**Comment by lego963** [comment]
recheck

**Review by anton-sidelnikov — APPROVED** [review]

**Comment by anton-sidelnikov** [comment]
regate

**Comment by anton-sidelnikov** [comment]
regate

**Pull request merged**